BitmovinPlayerSSAI/BitmovinPlayerSSAI/ContentView.swift

```diff
@@ -0,0 +1,105 @@
+import SwiftUI
+import BitmovinCollector
+import BitmovinPlayer
+import CoreCollector
+
+struct ContentView: View {
+    private let player: Player
+    private let playerViewConfig: PlayerViewConfig
+    private let ssaiAdProvider: SsaiInformationProvider = StaticSsaiInformationProvider()
+    private let bitmovinAdScheduler: BitmovinPlayerSsaiAdScheduler
+    private var sourceConfig: SourceConfig {
+        guard let streamUrl = URL(string: "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8") else {
+            fatalError("Invalid URL")
+        }
+
+        return SourceConfig(url: streamUrl, type: .hls)
+    }
+
+    init() {
+        // Create player configuration
+        let playerConfig = PlayerConfig()
+
+        // Set your player license key on the player configuration
+        playerConfig.key = "7ef40b69-f1c0-4c2c-8212-32b973204800"
```

Comment:
We should not use license keys in public samples.

Reply:
removed: https://github.com/bitmovin/bitmovin-analytics-collector-ios-samples/pull/8/commits/f0371544c819e9c3fc580b8c020c0f7bbe3ea86e